aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,373 @@
+.. _web-api-help-pages-using-swagger:
+
+ASP.NET Web API Help Pages using Swagger
+========================================
+
+By `Shayne Boyer`_
+
+Understanding the various methods of an API can be a challenge for a developer when building a consuming application. 
```

Comment:
Can we add a sentence here stating what Swagger is? Something like "Swagger is a machine readable representation of a RESTful API that enables support for interactive documentation, client SDK generation and discoverability."